config.template.yaml

```diff
@@ -4,11 +4,11 @@
 # user: username to authenticate as
 # password: password for user
 mysql:
-  server: 'localhost'
-  port: 3306
-  database: 'cytube3'
-  user: 'cytube3'
-  password: ''
+  server: '${MYSQL_HOST}'
```

Comment:
Is there any way you could make Docker use a separate template file?

All these Docker-friendly replace markers makes it a substantially greater pain in the ass for a regular human to edit the template to working order.